dbt_subprojects/dex/models/trades/sei/platforms/carbon_defi_sei_base_trades.sql

```diff
@@ -0,0 +1,57 @@
+{{
+    config(
+        schema = 'carbon_defi_sei',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{% set weth_address = '0x160345fc359604fc6e70e3c5facbde5f7a9342d8' %}
```

Comment:
I am not sure if this is the correct `WETH` address on sei:
https://seistream.app/contracts/0x160345fc359604fc6e70e3c5facbde5f7a9342d8

Reply:
yes, this is `weth`, but i think we actually want to use `WSEI` here as `SEI` is the native token to the chain, correct?

i think `weth` on `sei` is simply another erc20 that can be bridged across, but anything using `0xeee..` address would translate to `WSEI` for join to `prices.usd`

if we run both of these:
```
select targetToken, count(1)
from carbon_defi_sei.CarbonController_evt_TokensTraded
where targetToken = 0xE30feDd158A2e3b13e9badaeABaFc5516e95e8C7
group by targetToken
order by 2 desc

select targetToken, count(1)
from carbon_defi_ethereum.CarbonController_evt_TokensTraded
where targetToken = 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
group by targetToken
order by 2 desc
```

we see v little usage on the wrapped native token per chain, but if you remove the where clause, you can see the opposite side. on ethereum, lots of stablecoins and `0xeee..`. on sei, lots of stablecoins and `0xeee..`

Reply:
can we turn `weth_address` variable to a more generic `wrapped_native_token` variable?
then update sei to use `WSEI` [address](https://seitrace.com/token/0xE30feDd158A2e3b13e9badaeABaFc5516e95e8C7?chain=pacific-1)?
then we should double check our prices spells, ensure on sei chain, we have `WSEI` and `WETH` in tokens file